rules/windows/process_creation/win_susp_calc.yml

```diff
@@ -19,7 +19,7 @@ detection:
         Image|endswith: '\calc.exe'
     filter2:
         Image|contains: '\Windows\Sys'
```

Comment:
as path can be "C:\Windows\SysWOW64" or "C:\Windows\System32" can change  to be more accurate
`Image|contains: ':\Windows\Sys'`